### ISSUE TITLE
Illegal point sequence not detected

### ISSUE THREAD

**Issue by madig** [source-foundry/ufolint#132]
ufolint passes the following glif file:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<glyph name="test" format="2">
	<advance width="698"/>
	<outline>
		<contour>
			<point x="453" y="729" type="curve" smooth="yes"/>
			<point x="436" y="729"/>
			<point x="426" y="740"/>
			<point x="426" y="759" type="curve" smooth="yes"/>
			<point x="426" y="784"/>
			<point x="444" y="805"/>
			<point x="466" y="805" type="line" smooth="yes"/>
			<point x="483" y="805"/>
			<point x="493" y="794"/>
			<point x="493" y="775" type="curve" smooth="yes"/>
			<point x="493" y="750"/>
			<point x="475" y="729"/>
		</contour>
	</outline>
</glyph>
```

Note the `type="line"` that should be a curve. Loading the source with glifLib throws an error however.

**Comment by chrissimpkins** [comment]
Something appropriate upstream and exception caught here?

**Comment by madig** [comment]
Well upstream catches it on glif load but ufolint doesn't 🤔

**Comment by chrissimpkins** [comment]
That is either a bug here or something new that hasn't made it into the defined dependency versions.  We should be catching everything that raises on UFO reads with validation toggled on.
